From 9acc4e99a7df1d68aef504c3216973c658fd9dd5 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Tue, 18 Jul 2023 10:16:06 +0200
Subject: IWYU: missing include for round() call in tones.cc

Bug: chromium:957519
---
 cpp/palettes/tones.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cpp/palettes/tones.cc b/cpp/palettes/tones.cc
index 7b42646..a4e9487 100644
--- a/cpp/palettes/tones.cc
+++ b/cpp/palettes/tones.cc
@@ -16,6 +16,8 @@
 
 #include "cpp/palettes/tones.h"
 
+#include <math.h>
+
 #include "cpp/cam/cam.h"
 #include "cpp/cam/hct.h"
 
-- 
2.39.2

