From d1d0cea737e0b8fd1f33705972e67595cacbd483 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 13 Nov 2023 18:06:40 +0100
Subject: IWYU: missing include for std::string use in instrumentation.h

Bug: chromium:957519
---
 ruy/profiler/instrumentation.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ruy/profiler/instrumentation.h b/ruy/profiler/instrumentation.h
index c4df1e6..2b15ae3 100644
--- a/ruy/profiler/instrumentation.h
+++ b/ruy/profiler/instrumentation.h
@@ -19,6 +19,7 @@ limitations under the License.
 #ifdef RUY_PROFILER
 #include <cstdio>
 #include <mutex>
+#include <string>
 #include <vector>
 #endif
 
-- 
2.40.1

